api4/system.go

```diff
@@ -410,7 +410,27 @@ func pushNotificationAck(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	err := c.App.SendAckToPushProxy(ack)
-	if err != nil {
+	if ack.NotificationType == model.PUSH_TYPE_ID_LOADED {
+		if err != nil {
+			// Log the error only, then continue to fetch notification message
+			c.App.NotificationsLog.Error("Notification ack not sent to push proxy",
+				mlog.String("ackId", ack.Id),
+				mlog.String("type", ack.NotificationType),
+				mlog.String("postId", ack.PostId),
+				mlog.String("status", err.Error()),
+			)
+		}
+
+		msg, appErr := c.App.BuildFetchedPushNotificationMessage(ack.PostId, c.App.Session.UserId)
+		if appErr != nil {
+			c.Err = model.NewAppError("pushNotificationAck", "api.push_notification.id_loaded.fetch.app_error", nil, appErr.Error(), http.StatusInternalServerError)
```

Comment:
I'm getting this error when I used https://push-test.mattermost.com 
![Screen Shot 2019-11-19 at 1 28 17 AM](https://user-images.githubusercontent.com/5334504/69078250-6e38ff00-0a72-11ea-8a72-2d41794218b9.png) which doesn't have ID loaded push notification.

Reply:
This should be solved now